eng/common/pipelines/templates/steps/create-pull-request.yml

```diff
@@ -81,26 +86,8 @@ steps:
     filePath: ${{ parameters.ScriptDirectory }}/add-pullrequest-reviewers.ps1
     arguments: >
       -RepoOwner "${{ parameters.RepoOwner }}"
-      -RepoName "${{ parameters.RepoName }}"
+      -RepoName "$(RepoNameWithoutOwner)"
       -AuthToken "$(azuresdk-github-pat)"
       -GitHubUsers "$(${{ parameters.GHReviewersVariable }})"
       -GitHubTeams "$(${{ parameters.GHTeamReviewersVariable }})"
       -PRNumber "$(Submitted.PullRequest.Number)"
-
```

Comment:
@chidozieononiwu looks like your sync PR that added this got merged prematurely so the diffing is a little off. I told @sima-zhu to just go ahead and merge this PR as-is and we can do a clean-up sync after the current set of PR's are in. Can you please look at doing a full sync once you get your git patching changes in?